src/cmap/commands.ts

```diff
@@ -837,6 +838,12 @@ export class BinMsg {
     const promoteValues = options.promoteValues ?? this.opts.promoteValues;
     const promoteBuffers = options.promoteBuffers ?? this.opts.promoteBuffers;
     const bsonRegExp = options.bsonRegExp ?? this.opts.bsonRegExp;
```

Comment:
@durran @nbbeeken Just to double check, are we concerned at all with the `parse` method in the Response class or just the BinMsg? When do we hit the Response logic?

Reply:
We don't! Fortunately, we dropped the server versions that do not use OP_MSG so this condition in message_stream.ts:
```ts
ResponseType = messageHeader.opCode === OP_MSG ? BinMsg : Response;
```
always returns BinMsg, we could add the logic for completeness but that section of code is destined for deletion.